server/api/controllers/candidates.py

```
@@ -0,0 +1,379 @@
+            "educations": educations,
+            "experiences": experiences,
+            "user": {
+                "id": user.id,
+                "full_name": user.full_name,
+                "email": user.email,
+                "role": user.role,
+                "image": user.image,
+            }
+        }), 200
+        
+    except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)